patches/pgvector.patch

```diff
@@ -49,8 +49,12 @@ index 680789ba9044900eac9321844ee2a808a4a2ed12..41c5b709bcb2367ac8b8c498788ecac4
 +	}
 +
 +#ifdef NEON_SMGR
-+	smgr_end_unlogged_build(index->rd_smgr);
++	smgr_end_unlogged_build(RelationGetSmgr(index));
 +#endif
 +
  	FreeBuildState(buildstate);
  }
+ 
+-- 
```

Comment:
do we need this in the patch?

Reply:
There's no harm in keeping it. Also CI is really busy today, so I'd rather merge as is instead of rerunning the whole pipeline